x/lightclient/keeper/proof_test.go

```diff
@@ -0,0 +1,156 @@
+package keeper_test
+
+import (
+	"testing"
+
+	ethcommon "github.com/ethereum/go-ethereum/common"
+
+	ethtypes "github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/stretchr/testify/require"
+	"github.com/zeta-chain/zetacore/pkg/chains"
+	"github.com/zeta-chain/zetacore/pkg/proofs"
+	"github.com/zeta-chain/zetacore/pkg/proofs/ethereum"
+	"github.com/zeta-chain/zetacore/pkg/testdata"
+	keepertest "github.com/zeta-chain/zetacore/testutil/keeper"
+	"github.com/zeta-chain/zetacore/testutil/sample"
+	"github.com/zeta-chain/zetacore/x/lightclient/types"
+)
+
+// generateProof generates a proof and block header
+// returns the proof, block header, block hash, tx index, chain id, and tx hash
+func generateProof(t *testing.T) (*proofs.Proof, proofs.BlockHeader, string, int64, int64, ethcommon.Hash) {
```

Comment:
This might also be a good candidate for a sample; I feel any test data-related functions should be moved to the sample directory.

Reply:
Sounds good 60e57f36